Mathlib/Algebra/GroupWithZero/Units/Basic.lean

```diff
@@ -375,6 +375,7 @@ lemma div_eq_one_iff_eq (hb : b ≠ 0) : a / b = 1 ↔ a = b := hb.isUnit.div_eq
 lemma div_mul_cancel_right₀ (hb : b ≠ 0) (a : G₀) : b / (a * b) = a⁻¹ :=
   hb.isUnit.div_mul_cancel_right _
 
+set_option linter.deprecated false in
```

Comment:
General remark, shouldn't the linter automatically be off on declarations that are themselves deprecated? cc @digama0

Reply:
I asked about this at https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/can.20the.20.60deprecated.60.20attribute.20set.20.60linter.2Edeprecated.20false.60/near/437754423

Reply:
Nice! I hadn't seen this.